mteb/tasks/Classification/multilingual/MultilingualSentimentClassification.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification, MultilingualTask, TaskMetadata
+
+_LANGUAGES = {
+    "urd": ["urd-Arab"],
+    "vie": ["vie-Latn"],
+    "dza": ["dza-Arab"],
+    "tha": ["tha-Thai"],
+    "tur": ["tur-Latn"],
+    "slk": ["slk-Latn"],
+    "nor": ["nor-Latn"],
+    "spa": ["spa-Latn"],
+    "rus": ["rus-Cyrl"],
+    "mlt": ["mlt-Latn"],
+    "kor": ["kor-Hang"],
+    "ind": ["ind-Latn"],
+    "heb": ["heb-Latn"],
+    "jpn": ["jpn-Jpan"],
+    "ell": ["ell-Latn"],
+    "deu": ["deu-Latn"],
+    "eng": ["eng-Latn"],
+    "fin": ["fin-Latn"],
+    "hrv": ["hrv-Latn"],
+    "zho": ["zho-Hans"],
+    "cmn": ["cmn-Hans"],
+    "bul": ["bul-Cyrl"],
+    "eus": ["eus-Latn"],
+    "uig": ["uig-Hans"],
+    "bam": ["bam-Latn"],
+    "pol": ["pol-Latn"],
+    # "cym": ["cym-Latn"], # Do not handle this subset since it does not contain a train set required by the evaluation
```

Comment:
Could we potentially manually create a train test split via stratified sampling?

Reply:
I can use `train_test_split` for these `cym` since it contains 25k samples in the test set. 
Although for `hin` the number of samples is under 700 for train and validation combined, I'm not sure it's enough and I'm not comfortable with creating a test split from a train split for a benchmark

Reply:
> creating a test split from a train split for a benchmark

I feel the same way. Let's go with what you suggested for `cym`. For `hin` I understand the concern. Is it time consuming to see what performance we get from the just train set as is and eval on the validation set?